apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameterCollection.cs

```diff
@@ -140,6 +140,12 @@ public override void CopyTo(Array array, int index)
         }
 
         /// <inheritdoc />
+        IEnumerator<SpannerParameter> IEnumerable<SpannerParameter>.GetEnumerator()
+        {
+            return _innerList.GetEnumerator();
+        }
+
+        /// <inheritdoc />
         public override IEnumerator GetEnumerator() => _innerList.GetEnumerator();
```

Comment:
since its an override I dont believe I have a choice in the matter.
the base class defines it as implicit (unfortunately)